Fix spelling errors

YogoGit opened the pull request:
Simple spelling fixes.

YogoGit committed:
Fix spelling errors

redakker commented:
Hi! Thanks for contributing. Nice find. Your fix is right, but not in the right file. webcontent.h is a generated file during the build.
Please modify the source index.html in the html folder. Thx!

YogoGit committed:
Remove duplicate code

YogoGit committed:
Ensure the files are processed in the same order

YogoGit commented:
Updated to do it properly.  Note, I removed some code in pre_build.py and pre_build_web.py, and ensured the webcontent.h file is created the same way by ordering the file processing in the latter, as os.walk() does not guarantee order, and when I recreated the .h file, the order was completely different, and by using sort the contents were generated in the same order.

redakker commented:
Thanks! Ordering is okay, but unnecessary since it has no impact on the build process. It just minifies the file content. However you are right, if all builds are the same, it is better. :)

> Updated to do it properly. Note, I removed some code in pre_build.py and pre_build_web.py, and ensured the webcontent.h file is created the same way by ordering the file processing in the latter, as os.walk() does not guarantee order, and when I recreated the .h file, the order was completely different, and by using sort the contents were generated in the same order.
